docs/narr/MyProject/development.ini

```diff
@@ -1,6 +1,6 @@
 ###
 # app configuration
-# http://docs.pylonsproject.org/projects/pyramid/en/latest/narr/environment.html
+# http://docs.pylonsproject.org/projects/pyramid/en/1.7-branch/narr/environment.html
```

Comment:
Yeah, there's really no good option, kind of like voting for President.

If you're on the 1.5 branch, latest is not correct.  When we release 1.7, then latest will not be correct on the 1.6 branch docs.  And so on.

Additionally, when the scaffold generates a project, the actual version number appears in these URLs, never "latest".
```ini
###
# app configuration
# http://docs.pylonsproject.org/projects/pyramid/en/{{pyramid_docs_branch}}/narr/environment.html
###
```
1.7-branch is the least bad option.